frontend/src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import '../../assets/global.css';
```

Comment:
This should be ``import '../../assets/Global.css';``

Also is it possible to use absolute path imports similar to how we are doing it in the backend?